cli/cache_proxy/cache_proxy.go

```diff
@@ -31,7 +31,7 @@ var (
 )
 
 const (
-	queueBufferSize = 100
+	queueBufferSize = 1000
```

Comment:
if these are just digests, you could probably do 10K or more without worrying about memory

Reply:
Bumped to 10K.

As a side note, since the digests can have different sizes, I have mild concerns about creating a large backlog and having a background process that seems to be unexpectedly hogging upload speed, e.g. when trying to do a video call shortly after a `bb` build. We might want to make this based on the total size, rather than the number of uploads.